terminus/geometry/latlon.py

```diff
@@ -89,3 +89,28 @@ def delta_in_meters(self, other):
             return (delta_lat_in_meters, 0)
         else:
             return (delta_lat_in_meters, delta_lon_in_meters)
+
+    def middle_point(self, other):
+        # This method gives an approximation of the middle point between two LatLon objects, but it is not the 'real' midpoint.
```

Comment:
Use `'''` for block comments please. Also try to cut them at 80 char, otherwise the reader needs to scroll horizontally to see the whole sentence.